src/components/CommentInput/index.tsx

```diff
@@ -35,6 +35,16 @@ const CommentInput: FC<PropsType> = ({ type, uuid, source }) => {
     []
   );
 
+  const onSubmitEnter = useCallback(
+    (e: React.KeyboardEvent) => {
+      if (e.key === "Enter") {
```

Comment:
```suggestion
      if (!e.shiftKey && e.key === "Enter") {
```
이러면 줄바꿈을 할 수 없으니까
쉬프트키를 눌렀을때는 submit 하지 말고 엔터 되게 합시다.
줄바꿈은 필요할거 같네요

Reply:
아하! 고치고 올게유!

Reply:
근데 원래 줄바꿈은 대부분 shiftkey+enterkey 같이 누르면서 사용되니까 엔터만 조건해도 되지만 더 자세하기 위해 진근 님이 해 주신 식으로 뚝딱할게유